package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "pxbluecolors",
-    "version": "3.0.0",
+    "version": "3.0.1",
```

Comment:
I think you should make this 0.0.0 an change https://github.com/pxblue/colors/blob/dev/ui/package.json to 3.0.1

Reply:
You're right; this was dropped to 0.0.0 to prevent future confusion.